### PR TITLE
Bug - Fix issues for Ansible and benchmarks

### PR DESCRIPTION
**Pull request by abuccts**
__Description__

Fix issues for Ansible and benchmarks:
* Cleanup Ansible runner private data dir to avoid out of disk space issue when node number is large.
* Support both absolute and relative paths when fecth results.
* Use a deterministic image in Ansible test to avoid image update.
* Update logging format.
* Delete torch models and inputs after export.

### CODE REVIEW

File: tests/ansible/tests/test_deploy.yaml [hunk 1]
```diff
@@ -14,4 +14,4 @@
   vars:
     ssh_port: 12345
     output_dir: /tmp/test_ansible
-    docker_image: superbench/superbench
+    docker_image: superbench/superbench:v0.3.0-cuda11.1.1
```
Comment (cp5555) [inline tests/ansible/tests/test_deploy.yaml:17]: Do we need to change this?

Comment (abuccts) [reply tests/ansible/tests/test_deploy.yaml:17]: to avoid frequent image update and no space issue

**Review by yzygitzh — APPROVED** [review]

File: tests/ansible/tests/test_deploy.yaml (hunk 1 above)
Comment (abuccts) [reply tests/ansible/tests/test_deploy.yaml:18]: add comment, the image requires `sb` installed

Comment (cp5555) [reply tests/ansible/tests/test_deploy.yaml:18]: Got it, thanks!

**Review by cp5555 — APPROVED** [review]

**Pull request merged**